test/puppetlabs/puppetdb/query_eng_test.clj

```diff
@@ -46,16 +47,17 @@
 
 (deftest test-plan-cte
   (is (re-matches
-         #"WITH inactive_nodes AS \(SELECT certname FROM certnames WHERE \(deactivated IS NOT NULL AND deactivated > '\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d\.\d\d\dZ'\) OR \(expired IS NOT NULL and expired > '\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d\.\d\d\dZ'\)\), not_active_nodes AS \(SELECT certname FROM certnames WHERE \(deactivated IS NOT NULL OR expired IS NOT NULL\)\) SELECT table.foo AS \"foo\" FROM table WHERE \(1 = 1\)"
-         (plan->sql (map->Query {:projections {"foo" {:type :string
-                                           :queryable? true
-                                           :field :table.foo}}
-                      :alias "thefoo"
-                      :subquery? false
-                      :where (->BinaryExpression := 1 1)
-                      :selection {:from [:table]}
-                      :source-tables #{:table}})
-                    {:node-purge-ttl (parse-period "14d")}))))
+       #"WITH inactive_nodes AS \(SELECT certname FROM certnames WHERE \(deactivated IS NOT NULL AND deactivated > '\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d\.\d\d\dZ'\) OR \(expired IS NOT NULL and expired > '\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d\.\d\d\dZ'\)\), not_active_nodes AS \(SELECT certname FROM certnames WHERE \(deactivated IS NOT NULL OR expired IS NOT NULL\)\) SELECT table.foo AS \"foo\" FROM table WHERE \(\? = \?\)"
+       (-> {:projections {"foo" {:type :string
+                                 :queryable? true
+                                 :field :table.foo}}
+            :alias "thefoo"
+            :subquery? false
+            :where (->BinaryExpression := 1 1)
+            :selection {:from [:table]}
+            :source-tables #{:table}}
+           map->Query
+           (plan->sql {:node-purge-ttl (parse-period "14d")})))))
```

Comment:
I just threaded this while fixing the indentation.  The only substantive change here should be the change to the regex, i.e. `1 = 1` -> `? = ?`.